ui/js/actions/content.js

```diff
@@ -384,16 +384,17 @@ export function doFetchClaimsByChannel(uri, page) {
     });
 
     lbry.claim_list_by_channel({ uri, page: page || 1 }).then(result => {
-      const claimResult = result[uri],
-        claims = claimResult ? claimResult.claims_in_channel : [],
-        currentPage = claimResult ? claimResult.returned_page : undefined;
+      const claimResult = result[uri] || {};
+      const { claims_in_channel, returned_page } = claimResult;
+      const claims = claims_in_channel || [];
+      const page = returned_page || undefined;
```

Comment:
@btzr-io technically this overwrites an in scope variable, which ought to be avoided.

Reply:
(I fixed this before I merged)